spec/requests/api/querying_spec.rb

```diff
@@ -91,6 +91,16 @@ def create_vms_by_name(names)
       expect_query_result(:vms, 3, 3)
       expect_result_resources_to_match_hash([{"name" => "a"}, {"name" => "B"}, {"name" => "c"}])
     end
+
+    it "supports sorting with physical attributes" do
+      FactoryGirl.create(:vm_vmware, :name => "vmware_vm")
```

Comment:
The vendor is specified by the type specific fixtures, :vm_vmware or :vm_redhat, I can respecify here to minimize confusion.  no problem.